content/providers/01-ai-sdk-providers/26-fireworks.mdx

```diff
@@ -116,6 +116,7 @@ const model = fireworks.completion('accounts/fireworks/models/firefunction-v1');
 
 | Model                                                      | Image Input         | Object Generation   | Tool Usage          | Tool Streaming      |
 | ---------------------------------------------------------- | ------------------- | ------------------- | ------------------- | ------------------- |
+| `accounts/fireworks/models/deepseek-r1`                    | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Cross size={18} /> |
```

Comment:
shouldn't it be cross for tool usage? and also for object generation?

Reply:
You're right, I've crossed it for Tool usage.
Should I do it for Object generation as well? I'm not sure how to find if that's the case